package.json

```diff
@@ -16,6 +16,8 @@
     "dev": "tsdx watch",
     "prebuild": "npm run test",
     "build": "tsdx build",
+    "prepublish": "npm run build",
```

Comment:
I think that this line needs to go away

Reply:
Why?

Reply:
Maybe prepublishOnly is better?

Reply:
we removed this in hospitalrun/components#48

Reply:
My fault!

Reply:
no worries :)